packages/shorebird_cli/lib/src/commands/patch/ios_framework_patcher.dart

```diff
@@ -108,13 +133,17 @@ class IosFrameworkPatcher extends Patcher {
       throw ProcessExit(ExitCode.software.code);
     }
     try {
+      if (splitDebugInfoPath != null) {
+        Directory(splitDebugInfoPath!).createSync(recursive: true);
+      }
       await artifactBuilder.buildElfAotSnapshot(
         appDillPath: buildResult.kernelFile.path,
         outFilePath: p.join(
           shorebirdEnv.getShorebirdProjectRoot()!.path,
           'build',
           'out.aot',
         ),
+        additionalArgs: [...splitDebugInfoArgs],
```

Comment:
```suggestion
        additionalArgs: splitDebugInfoArgs,
```

Reply:
I intentionally did this so that it was easier to add more args in the future but happy to adjust